### PR TITLE
Scan existing schedules just in case to detect orphans

### PR DESCRIPTION
**Pull request by pavelzubarev**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ skip_glob = '^((?!py$).)*$'    # this makes sort all Python files
 
 [tool.poetry.dependencies]
 python = "^3.7"
-cognite-sdk-experimental = "^0.45.0"
+cognite-sdk-experimental = "^0"
```
Comment (haakonvt) [inline pyproject.toml:20]: wut

Comment (pavelzubarev) [reply pyproject.toml:20]: just in case it gets all the shit when you do `poetry update` major included

**Review by haakonvt — COMMENTED** [review]
Remove the poetry changes? Then lets merge 😄

File: pyproject.toml (hunk 1 above)
Comment (haakonvt) [reply pyproject.toml:20]: I don't understand - we need at least a specific version, right?

Comment (haakonvt) [reply pyproject.toml:20]: *minimum version

Comment (pavelzubarev) [reply pyproject.toml:20]: image

**Review by haakonvt — APPROVED** [review]

**Pull request merged**